manual_correction.py

```diff
@@ -584,6 +590,8 @@ def ask_if_modify(fname_out, fname_label, do_labeling_always=False):
         create_empty_mask = False
 
     # If the output file does not exist, copy it from label folder
+        print(os.path.isfile(fname_label))
+        print('fname_label', fname_label)
```

Comment:
@rohanbanerjee, these two prints might be removed, right? I guess they were used for debugging.